src/lms-ds-loader/tests/test_argparser.py

```diff
@@ -0,0 +1,352 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import pytest
+
+from lms_ds_loader.argparser import parse_arguments
+from lms_ds_loader.constants import Constants
+
+
+class Test_parse_arguments:
+
+    PATH = "./lms_udm_files"
+    SERVER = "localhost"
+    DB_NAME = "EdFi_ODS"
+    USERNAME = "my_user"
+    PASSWORD = "my_password"
+    PORT = 1234
+
+    def path_args(self):
+        return ["--csvpath", self.PATH]
+
+    def engine_args(self, engine):
+        return ["--engine", engine]
+

```

Comment:
This could work for the methods that don't require parameters. So when you call them you can do it like:
`object.server_args`

```suggestion

@property
```